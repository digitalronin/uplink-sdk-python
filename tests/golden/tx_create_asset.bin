00000000: 03 EB 58 8A 67 D5 F2 D0  08 A3 96 E1 AB 6E 89 AB  ..X.g........n..
00000010: 2C 85 1E 06 77 F0 B1 63  37 90 55 DD 08 32 74 C9  ,...w..c7.U..2t.
00000020: B2 EF 00 04 74 65 73 74  00 00 00 00 00 00 03 E8  ....test........
00000030: 00 01 00 05 54 6F 6B 65  6E 00 08 44 69 73 63 72  ....Token..Discr
00000040: 65 74 65                                          ete