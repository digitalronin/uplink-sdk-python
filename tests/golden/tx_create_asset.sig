r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 85958517410874828515647930073382159412224479333145965657989388575420205184773
