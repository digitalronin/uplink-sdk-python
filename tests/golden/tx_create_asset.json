{
    "header": {
        "tag": "TxAsset",
        "contents": {
            "tag": "CreateAsset",
            "contents": {
                "assetName": "test",
                "supply": 1000,
                "issuer": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "reference": "Token",
                "assetType": {
                    "tag": "Discrete",
                    "contents": 0
                }
            }
        }
    },
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE00NTk4NTI0OTM0OTY3MDUzMDI4ODYzODI2NTg1NTQxNzQxNDExMjk1MDYzNTUwNDYyODkyNDk4MzczODc3MzYyMDM5MzEzMzEyNTc2Ng==",
    "timestamp": 1231006505,
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "to": null
}